src/Compilers/CSharp/Portable/Symbols/TypeSymbolExtensions.cs

```diff
@@ -705,14 +694,36 @@ private static bool IsTypeLessVisibleThan(TypeSymbol type, Symbol sym, ref HashS
                 type = canDigThroughNullable ? next.NullableUnderlyingTypeOrSelf : null;
             }
 
-            TypeSymbol? visitType(TypeWithAnnotations typeArg) => VisitType(
+            static TypeSymbol? visitFunctionPointerType(FunctionPointerTypeSymbol type, Func<TypeWithAnnotations, T, bool, bool>? typeWithAnnotationsPredicate, Func<TypeSymbol, T, bool, bool>? typePredicate, T arg, bool useDefaultType, bool canDigThroughNullable)
+            {
+
+                MethodSymbol currentPointer = type.Signature;
```

Comment:
Consider removing the blank line.

Reply:
I'll address this in a follow up.

---
In reply to: [421810459](https://github.com/dotnet/roslyn/pull/44040#discussion_r421810459) [](ancestors = 421810459)